src/__tests__/Interpreter.test.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Addition,
+  Multiplication,
+  Negative,
+  Group,
+  Num,
+  Division,
+} from '../Expression'
+import { Interpreter } from '../Interpreter'
+import { parse } from '../Parser'
+import { scan } from '../Scanner'
+
+test('addition works correctly', () => {
```

Comment:
Can you add some tests for grouping? And longer expressions like:

`3 + 4 + 5 * 8`

Reply:
Added a bunch more tests of varying complexity